### DISCUSSION TITLE
[WP2861 Discussion]: Might be a little... bare

### DISCUSSION THREAD

**Discussion by Artoria2e5** [wikipathways/pathway-curation-tasks#35] [General]
### Discussion

WP2861 is a little ~~bear~~ bare compared to other examples under the [ETC category](https://www.wikipathways.org/browse/table.html?Pathway%20Ontology=electron%20transport%20chain%20pathway). Compared to WP111 (knowing full well a chloroplast is not a mitochondrion), I think we need:

* Protons going through the ATPase "downward" into the stroma.
* A quinone doing stuff between the two PS boxes.
* b6f making an appearance. And plastocyanin.
* Some text above the PSII and PSI boxes to remind people that these things are photosystems.
* Maybe some shortening of the names in the PS boxes?

The pathway is also part of WP2859. In that pic it looks okay-ish, considering it's not the main character.

### Type of revision

- [ ] Major
- [ ] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
Great suggestions @Artoria2e5. Here are some source materials that might help in imporving this pathway:

Published pathway figures from the [Pathway Figure OCR](https://pfocr.wikipathways.org/) project: 
- https://pfocr.wikipathways.org/figures/PMC9545066__TPJ-111-642-g003.html
- https://pfocr.wikipathways.org/figures/PMC8505746__fpls-12-748204-g001.html

Other sources:

- https://www.researchgate.net/figure/Schematic-Representation-of-Photosynthetic-Electron-Transport-Photosystem-I-PSI_fig2_337850288
- https://www.researchgate.net/figure/The-photosynthetic-electron-transfer-chain-and-the-reduction-of-chloroplastic-TRXs-In_fig2_259268699
- https://en.wikipedia.org/wiki/Light-dependent_reactions
